fix: Update Manual.md

Briancoughlin opened the pull request:
Putting in the corrections to fix the index.md issue with the  0.1.0 package as raised in the forums.

https://forum.unity.com/threads/unity-mlapi-manual-info-page.1167625/

Briancoughlin committed:
Create Index.md

Briancoughlin committed:
Update Index.md

JesseOlmer commented:
A few things:
- Can you have this PR target `develop` branch instead? We cannot ship a hotfix for 0.1.0
- Why do we need 2 identical files? Does 1 not suffice?
- Do we have any plans to support redirects in the future if/when this happens again? It'd be a shame if all our package links were dead in a year for older versions of the package.
- Any way we can have these links point directly to the matching documentation version (0.1.0 instead of whatever latest/default ends up being)? Should we?

Briancoughlin commented:
- Can you have this PR target develop branch instead? We cannot ship a hotfix for 0.1.0. ( Done)
- Why do we need 2 identical files? Does 1 not suffice? (I think we only need `Index.md` I was just being cautious)
- Do we have any plans to support redirects in the future if/when this happens again? It'd be a shame if all our package links were dead in a year for older versions of the package. (it shouldn't happen again.This file is what the package manager looks for so as long as we keep it up to date and named correctly the issue should not happen again
- Any way we can have these links point directly to the matching documentation version (0.1.0 instead of whatever latest/default ends up being)? Should we? ( not sure on this one.......)

Commit:
docs: Rename Manual.md to Index.md and fix links

Commit:
docs: Fix Index.md to be latest develop version

Rebase from master branch had old data in Index.md, updated to match and pulled over the URL fix.

JesseOlmer reviewed (approved):
lgtm

Commit:
Merge branch 'develop' into Briancoughlin--fixing-the-index.md-file
